app/models/markdown_record/content_fragment.rb

```diff
@@ -19,12 +21,17 @@ def self.new_association(base_filters = {}, search_filters = {})
       MarkdownRecord::Association.new(base_filters, search_filters).fragmentize
     end
 
+    def find_relative(relative_id)
+      real_id = Pathname.new(id).parent.join(relative_id).to_s
+      self.class.find(real_id)
+    end
+
```

Comment:
Added this find method to help with finding content fragments relative to another